java/query/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -330,12 +336,14 @@ public void shouldSetStatusOfQueryToCOMPLETEDWhenOnlyOneSubQueryIsCreated() thro
                 .queryId("abc")
                 .regions(List.of(new Region(List.of(range1, range2, range3))))
                 .build();
+
         processQuery(query);
+        processLeafPartitionQuery();
 
         // Then
         assertThat(queryTracker.getAllQueries())
-                .usingRecursiveFieldByFieldElementComparatorIgnoringFields("lastUpdateTime", "expiryDate")
-                .containsExactly(trackedQuery()
+                .usingRecursiveFieldByFieldElementComparatorIgnoringFields("lastUpdateTime", "expiryDate", "subQueryId")
+                .contains(trackedQuery()
```

Comment:
This doesn't make any assertion about what else is in the list or how long it is. We can match how the other tests assert on the result of queryTracker.getAllQueries here.

It looks like the difference is that there's now a subquery as well. We could use containsExactlyInAnyOrder and add the sub query to the expected values list.

Reply:
Added subquery to the assertion